.github/workflows/build-documentation.yml

```diff
@@ -0,0 +1,22 @@
+name: build-documentation
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - name: Install prerequisites
+      run: |
+          python -m pip install sphinx==3.4.0 sphinx-prompt sphinx_substitution_extensions sphinx_book_theme
```

Comment:
Consider using `requirements.txt` instead of explicit list of requirements.

Reply:
Updated